src/mod/codecs/mod_opus/mod_opus.c

```diff
@@ -1412,6 +1426,7 @@ SWITCH_MODULE_LOAD_FUNCTION(mod_opus_load)
 											 switch_opus_destroy);	/* deinitalize a codec handle using this implementation */
 
 		codec_interface->implementations->codec_control = switch_opus_control;
+		codec_interface->implementations->vad = switch_opus_vad;
```

Comment:
you should add SCC_VAD to switch_codec_control_command_t, instead of adding "vad" here, in switch_codec_implementation.

Reply:
https://github.com/signalwire/freeswitch/pull/2108/files  - we added 2 getters to help you with calling switch_core_codec_control() from JB.